tests/GovUk.FrontEnd.AspNetCore.TestCommon/AssertEx.cs

```diff
@@ -78,7 +78,10 @@ public static void HtmlEqual(
                 DiffConverter.Append(diff, sb);
             }
 
-            throw new XunitException(sb.ToString());
+            throw new XunitException("---\nActual: \n---\n" + actual.Replace(">", ">\n").Replace("<", "\n<").Replace(">\n\n<", ">\n<").Trim() + "\n\n" +
+                                     "---\nExected: \n---\n" + expected + "\n\n" +
+                                     "---\n" +
+                                     "" + sb.ToString());
```

Comment:
The HTML diff is very useful/required to flag where the differences are, but it only gets so far if not able to visualise the broader context i.e. the syntax tree / html string it is referring to (a-la "stop at the third red house on the left").

I have found it useful to be able to quickly reference expected/actual HTML, to then try and figure out whether it is the container versus a nested element (or both) whose behaviour is wrong.

It's a bit of a hacky solution and might not be desirable to merge this, however?